# HG changeset patch
# User Ben Newman <bnewman@mozilla.com>
# Date 1258408982 28800
# Node ID 439a83c3eb8eabfc2ded69a48ec27a1b203a1ce8
# Parent  dbd8d7c3ac8fc85f38a95077d4e593fe910cf469
imported patch hsivonen/shim-between-514661-and-515338.diff

diff --git a/parser/html/nsHtml5TreeOperation.cpp b/parser/html/nsHtml5TreeOperation.cpp
--- a/parser/html/nsHtml5TreeOperation.cpp
+++ b/parser/html/nsHtml5TreeOperation.cpp
@@ -165,17 +165,17 @@ nsHtml5TreeOperation::Perform(nsHtml5Tre
                                         localName, 
                                         static_cast<PRUint8>(nsIDOMMutationEvent::ADDITION),
                                         stateMask);
         }
       }
       return rv;
     }
     case eTreeOpDoneAddingChildren: {
-      mNode->DoneAddingChildren(aBuilder->HaveNotified(mNode));
+      mNode->DoneAddingChildren(PR_FALSE);
       return rv;
     }
     case eTreeOpDoneCreatingElement: {
       mNode->DoneCreatingElement();
       return rv;
     }
     case eTreeOpUpdateStyleSheet: {
       aBuilder->UpdateStyleSheet(mNode);
